source/common/quic/envoy_quic_client_connection.cc

```diff
@@ -164,7 +167,7 @@ void EnvoyQuicClientConnection::maybeMigratePort() {
   probeWithNewPort(peer_address(), quic::PathValidationReason::kPortMigration);
 }
 
-void EnvoyQuicClientConnection::probeWithNewPort(const quic::QuicSocketAddress& peer_address,
+void EnvoyQuicClientConnection::probeWithNewPort(const quic::QuicSocketAddress& peer_addr,
```

Comment:
nit: why the rename?

Reply:
b/c we have to call the `peer_address()` member function on L186 and if we keep the variable name `peer_address`, the compiler gets confused and thinks we're trying to invoke a function on the `peer_address` variable instance